CRM/Core/BAO/File.php

```diff
@@ -13,8 +13,6 @@
  *
  * @package CRM
  * @copyright CiviCRM LLC https://civicrm.org/licensing
- * $Id$
```

Comment:
Note I only included one file to keep this readable since I *think* the one below is the 'source'